README.md

```diff
@@ -5,6 +5,18 @@ Monorepo for the [WordPress Performance Group](https://make.wordpress.org/core/t
 
 [Learn more about the Performance Lab plugin.](https://make.wordpress.org/core/2022/03/07/the-performance-lab-plugin-has-been-released/)
 
+## Quick Start
+In order to get started contributing to the project you have to follow the next steps:
+1. Fork the repository.
+2. Clone the fork locally.
+3. Run `composer install` in the project folder.
+4. Run `npm install` in the project folder.
+5. Start the development environment by running `npm run wp-env start`.
+6. Login using `admin` and `password`.
```

Comment:
I think we should also say something about where the user needs to go in order to log in. Maybe like this?
```suggestion
5. Start the development environment by running `npm run wp-env start`.
6. Go to `http://localhost:8888/wp-admin/` in your browser.
7. Log in using `admin` and `password`.
```